eras/babbage/impl/cardano-ledger-babbage.cabal

```
@@ -73,7 +73,7 @@ library
         cardano-crypto-class,
         cardano-data >=1.2,
         cardano-ledger-allegra ^>=1.5,
-        cardano-ledger-alonzo ^>=1.9,
+        cardano-ledger-alonzo ^>=1.10,
```

We can be a bit more lax
```suggestion
        cardano-ledger-alonzo >=1.9 && < 1.11,
```